docs/management/action-types.asciidoc

```diff
@@ -78,6 +78,11 @@ a| <<xmatters-action-type,xMatters>>
 a| <<torq-action-type,Torq>>
 
 | Trigger a Torq workflow.
+
+a| <<gen-ai-action-type,Generative AI>>
+
+| Send a request to OpenAI
```

Comment:
The connector itself refers to "generative AI" more generally. Is there a reason for the mismatch?
```suggestion
| Send a request to OpenAI.
```

Reply:
I guess with the title being Generative AI, I saw this as a description. Don't want to describe Generative AI by saying Generative AI... does that make sense?

- will add period here